news-explorer-frontend/src/utils/NewsApi.js

```diff
@@ -0,0 +1,42 @@
+const _checkResponse = (res) => {
```

Comment:
`_checkResponse` 

no need to duplicate `_checkResponse` 3 times in each file. You need to define it once and import to the file you need